src/services/types.ts

```diff
@@ -12,7 +12,7 @@ export type User = {
   email: string;
   id: number;
   password: string;
-  fullName: string;
+  full_name: string;
```

Comment:
Why are we changing this type?

Reply:
changed back the type. will make the changes in a new function